### PR TITLE
Add decapsulate-header and network-instance leafs for nexthop.

### PR DESCRIPTION
**Pull request by nflath**

### CODE REVIEW

**Comment by robshakir** [comment]
Hi Nathaniel -- each of the files within the `aft/` directory need to have the same version, can you copy the `openconfig-version` and `revision` statements into each file please?

Thanks!

**Comment by nflath** [comment]
Done

File: release/models/aft/openconfig-aft-common.yang [hunk 1]
```diff
@@ -347,6 +353,26 @@ submodule openconfig-aft-common {
...
+        "When forwarding a packet to the specified next-hop, the local
+         system performs a decapsulation of the packet - removing the
+         specified header type. In the case that no next-hop is
+         specified, the packet header is removed, and a subsequent
+         forwarding lookup is performed on the packet encapsulated
+         within the header, matched within the relevant AFT within the
+         specified network-instance.";
+    }
+
+    leaf network-instance {
```
[the remark this answers is not in the document]
Comment (robshakir) [reply release/models/aft/openconfig-aft-common.yang:368]: I think this was just replaced with the augmented one -- so likely we can back this out, and then just merge the decapsulation header.

[the remark this answers is not in the document]
Comment (robshakir) [reply release/models/aft/openconfig-aft-common.yang]: I made this change in Nathaniel's upstream fork.

**Review by robshakir — APPROVED** [review]

**Comment by robshakir** [comment]
Confirming the tree diff to demonstrate it's just the `decapsulate-header`:

```
robjs@penguin:~/openconfig/public/release/models/aft$ diff -prN /tmp/tree /tmp/nflath-trrobjs@penguin:~/openconfig/public/release/models/aft$ diff -prN /tmp/tree /tmp/nflath-tree 
*** /tmp/tree   2022-06-16 08:38:40.649839972 -0700
--- /tmp/nflath-tree    2022-06-16 08:37:02.973835495 -0700
*************** module: openconfig-network-instance
*** 1239,1244 ****
--- 1239,1245 ----
          |                          |  +--ro pop-top-label?             boolean
          |                          |  +--ro pushed-mpls-label-stack*   oc-mplst:mpls-label
          |                          |  +--ro encapsulate-header?        oc-aftt:encapsulation-header-type
+         |                          |  +--ro decapsulate-header?        oc-aftt:encapsulation-header-type
          |                          |  +--ro origin-protocol?           identityref
          |                          |  +--ro counters
          |                          |     +--ro out-pkts?             oc-yang:counter64
*************** module: openconfig-network-instance
*** 1515,1520 ****
--- 1516,1522 ----
          |        |  +--ro pop-top-label?               boolean
          |        |  +--ro pushed-mpls-label-stack*     oc-mplst:mpls-label
          |        |  +--ro encapsulate-header?          oc-aftt:encapsulation-header-type
+         |        |  +--ro decapsulate-header?          oc-aftt:encapsulation-header-type
          |        |  +--ro origin-protocol?             identityref
          |        |  +--ro lsp-name?                    string
```

**Pull request merged**